everyone-on-support.md

```diff
@@ -0,0 +1,33 @@
+# Prologue
+
+Everyone on Support is a regular activity wherein all full-time members are called upon to do a Support shift once every month (this includes customer and tech support). Trialists and part-time team members are excluded from this ritual.
+
+EOS is not designed to make everyone support or to be pulled from regular work to do support. The main reason is that a new set of eyes can help find issues and opportunities in current processes. A developer might see something that is not obvious to non-dev. A marketer might see a great opportunity that was dismissed till then.
+
+In the end, we are working for our customers, and having everyone involved in the conversation will help improve their experience and make them happy. And happy customers are happy to spend money with us.
+
+# Mechanics
+
+## I. Instructions
+
+* We use GrooveHQ for handling support emails. For EOS, we'll have one account which you'll need to personalize when you're on duty. Login credentials are on 1Password. Go to Settings and update the Name field.
+* Everyone has Staff access to the EBN app.
+* GrooveHQ integrates with Slack so we get a live stream of emails and responses. You can optionally join #support-emails channel.
+* You'll need to brush up on proper support decorum. See our Handbook.
+* All support documents can be found in Intra Easy Blog Networks folder. Make sure you go through them regularly and suggest improvement where needed.
+
+## II. Frequency
+
+EOS will be once every month. 
+
+Since we're on Sprints, it's expected that Team Capacity will be affected and duly adjusted.
+
+## III. Schedule
+
+Each one will set their own schedule and keep with it. Set a date and update the calendar in Intra.
```

Comment:
"Set a date and [update the calendar in Intra](https://intra.niteoweb.com/operations/everyone-on-support/view)."